tests/migrations/test_commands.py

```diff
@@ -546,6 +546,48 @@ def test_makemigrations_handle_merge(self):
         self.assertIn("Branch 0002_conflicting_second", output)
         self.assertIn("Created new merge migration", output)
 
+    def test_makemigration_merge_dry_run(self):
+        """
+        Makes sure that makemigrations respects --dry-run option when fixing migration conflicts.
```

Comment:
Do you mind prefixing the comment with the issue number: `#24427 - Makes sure ...`